yt_dlp/extractor/ard.py

```diff
@@ -630,13 +632,33 @@ def _real_extract(self, url):
     }
   }
 }''' % (client, video_id),
-            }).encode(), headers={
-                'Content-Type': 'application/json'
-            })['data']['playerPage']
-        title = player_page['title']
+                }).encode(), headers={
+                    'Content-Type': 'application/json'
+                })['data']['playerPage']
+
+            raw_player_page = player_page
+        except ExtractorError as e:
+            if not isinstance(e.cause, HTTPError) or e.cause.status != 503:
+                raise
+            else:  # try to get data from website API endpoint as public gateway failed with 503
+                player_page = self._download_json(
+                    f'https://api.ardmediathek.de/page-gateway/pages/ard/item/{video_id}', display_id, headers={
+                        'Content-Type': 'application/json'
+                    }, note='Downloading fallback JSON metadata')
```

Comment:
since this is not a POST request, the `content-type` header is likely unnecessary?
```suggestion
                    f'https://api.ardmediathek.de/page-gateway/pages/ard/item/{video_id}', display_id,
                    note='Downloading fallback JSON metadata')
```